dashboard/dashboard-session-sso.md

```diff
@@ -149,3 +149,51 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
     ![操作示例 3](/media/dashboard/dashboard-session-sso-okta-info-3.png)
 
 至此，TiDB Dashboard 已被配置为使用 Okta 进行 SSO 登录。
+
+## 示例二：使用 Auth0 进行 TiDB Dashboard SSO 登录认证
+
+和 Okta 类似，[Auth0](https://www.okta.com/) 也可以提供 OIDC SSO 的身份认证服务。
+
+### 步骤一：配置 Auth0
+
+1. 访问 Auth0 管理后台。
+
+2. 点击左侧边栏的 **Applications** > **Applications**。
+
+3. 点击 **Create Application**，在弹出窗口中输入 Name，比如 "TiDB Dashboard"，application type 选择 "Single Page Web Application"。
+
+    ![Create Application](/media/dashboard/dashboard-session-sso-auth0-create-app.png)
+
+4. 点击 **Settings** 栏。
+
+    ![Settings](/media/dashboard/dashboard-session-sso-auth0-settings-1.png)
+
+5. **Allowed Callback URLs** 字段填写如下内容：
+
+    ```
+    http://DASHBOARD_IP:PORT/dashboard/?sso_callback=1
+    ```
+
+    以上内容中，将 `DASHBOARD_IP:PORT` 替换为你在浏览器中实际访问 TiDB Dashboard 所使用的域名（或 IP）及端口。
+
+    此字段可以填入多个值，用逗号分隔，用于不同的环境，比如用于开发环境则可以加上 `http://localhost:3001/dashboard/?sso_callback=1`。
```

Comment:
We should not expose Develop knowledges to end users. This will just confuse readers.

Reply:
Ok, let me remove it.

Reply:
Let me think, but this doc is also for the developers? I read this doc to develop the SSO feature.

Reply:
Done.

Reply:
It is a user manual. A user manual can be useful for developers. But it doesn't mean that it should contain developer specific contents.